src/sentry/notifications/manager.py

```diff
@@ -72,74 +33,57 @@ def get_settings(
         self,
         provider: ExternalProviders,
         type: NotificationSettingTypes,
-        user=None,
-        team=None,
-        project=None,
-        organization=None,
-    ):
+        user: Optional[Any] = None,
+        team: Optional[Any] = None,
+        project: Optional[Any] = None,
+        organization: Optional[Any] = None,
+    ) -> NotificationSettingOptionValues:
         """
         In this temporary implementation, always read EMAIL settings from
         UserOptions. One and only one of (user, team, project, or organization)
         must not be null. This function automatically translates a missing DB
         row to NotificationSettingOptionValues.DEFAULT.
-        :param provider: ExternalProviders enum
-        :param type: NotificationSetting.type enum
-        :param user: (optional) A User object
-        :param team: (optional) A Team object
-        :param project: (optional) A Project object
-        :param organization: (optional) An Organization object
-        :return: NotificationSettingOptionValues enum
         """
-        user_id_option = getattr(user, "id", None)
-        scope_type, scope_identifier = _get_scope(
-            user_id_option, project=project, organization=organization
-        )
-        target = _get_target(user, team)
-
-        value = (  # NOQA
-            self.filter(
-                provider=provider.value,
-                type=type.value,
-                scope_type=scope_type,
-                scope_identifier=scope_identifier,
-                target=target,
-            ).first()
-            or NotificationSettingOptionValues.DEFAULT
+        from sentry.models.useroption import UserOption
```

Comment:
why was this moved down here? import issues?

Reply:
Yeah trying to prevent problems with circular imports.